package.json

```diff
@@ -115,12 +115,13 @@
     "wait-on": "^5.2.0"
   },
   "scripts": {
-    "start": "RADICLE_UPSTREAM_EXPERIMENTAL=false yarn start:app",
-    "start:experimental": "RADICLE_UPSTREAM_EXPERIMENTAL=true yarn start:app",
-    "start:app": "run-p --race rollup:watch proxy:start electron:start",
-    "start:release": "run-p --race rollup:watch proxy:start:release electron:start",
-    "start:experimental:release": "RADICLE_UPSTREAM_EXPERIMENTAL=true yarn start:release",
-    "start:test": "run-p --race rollup:watch proxy:start:test electron:start",
+    "start": "RADICLE_UPSTREAM_EXPERIMENTAL=false RADICLE_UPSTREAM_PROXY_PATH=../proxy/target/release/radicle-proxy yarn run:release:proxy",
+    "start:dev": "RADICLE_UPSTREAM_EXPERIMENTAL=false RADICLE_UPSTREAM_PROXY_PATH=../proxy/target/debug/radicle-proxy yarn run:dev:proxy",
+    "start:experimental": "RADICLE_UPSTREAM_EXPERIMENTAL=true RADICLE_UPSTREAM_PROXY_PATH=../proxy/target/release/radicle-proxy yarn run:release:proxy",
+    "start:experimental:dev": "RADICLE_UPSTREAM_EXPERIMENTAL=true RADICLE_UPSTREAM_PROXY_PATH=../proxy/target/debug/radicle-proxy yarn run:dev:proxy",
+    "run:test": "run-p --race rollup:watch proxy:start:test electron:start",
+    "run:dev:proxy": "yarn proxy:build && yarn run-p --race rollup:watch electron:start",
+    "run:release:proxy": "yarn proxy:build:release && yarn run-p --race rollup:watch electron:start",
```

Comment:
I’m getting more and more confused by the number of commands and what they do. I have a couple of suggestions.
1. Eleminate the `*:experimental:*` commands. They can easily be replicated by setting the environment variable. E.g. `RADICLE_UPSTREAM_EXPERIMENTAL=true yarn:start`
2. Some commands, e.g. `run:dev:proxy` don’t work if you call them directly. Either we remove them and inline their code or we clearly mark them as internal by changing their name.
3. Let’s try to make the naming more consistent. It’s hard to disinguish `start:dev` from `run:dev:proxy`. And the `proxy` part is confusing because it really runs more than the proxy.

Reply:
Could we also update the docs in `DEVELOPMENT.md`?

Reply:
I agree that it needs cleaning up. I regularly forget about the very useful `yarn start:test`. I think it is outside of the scope of this PR though.

Reply:
Good points, I could take it over and do it for this PR.

Reply:
@geigerzaehler addressed all points, PTAL!